shared/actions/chat2/index.tsx

```diff
@@ -3433,15 +3433,13 @@ const prepareFulfillRequestForm = (
 
 const addUsersToChannel = async (action: Chat2Gen.AddUsersToChannelPayload, logger: Saga.SagaLogger) => {
   const {conversationIDKey, usernames} = action.payload
+
   try {
     await RPCChatTypes.localBulkAddToConvRpcPromise(
       {convID: Types.keyToConversationID(conversationIDKey), usernames},
       Constants.waitingKeyAddUsersToChannel
     )
-    return [
-      RouteTreeGen.createClearModals(),
-      Chat2Gen.createNavigateToThread({conversationIDKey, reason: 'addedToChannel'}),
```

Comment:
Removed this, in both places where it's used, we no longer need to navigate to that channel
1. Used in info panel in a channel (here, we're already at the channel)
2. Used in Teams > Channel page, where we want to stay on that page